plugin/web/extensions/sidebar/core/components/hooks.ts

```diff
@@ -209,6 +209,19 @@ export default () => {
       payload: { addedDatasets: selectedIds, rawCatalog },
     });
   }, [rawCatalog, project.selectedDatasets]);
+
+  const handleClose = useCallback(() => {
+    postMsg({ action: "popupClose" });
+  }, []);
+
+  const handleDatasetAdd = useCallback(
+    (dataset: CatalogItem) => {
+      handleProjectDatasetAdd(dataset as CatalogRawItem);
+      handleClose();
+    },
+    [handleClose, handleProjectDatasetAdd],
+  );
```

Comment:
Can you keep these in the mobileDropdown? They are mobile specific and there is no need for them to be here (since you import handleProjectDatasetAdd with this hooks file in to the mobileDropdown anyways)

Reply:
Done!

Reply:
Done!